PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetEnergyFlow.h

```diff
@@ -67,7 +67,7 @@ class AliAnalysisTaskEmcalJetEnergyFlow: public AliAnalysisTaskEmcalJet {
 	void                    DoClusterLoop()                         ; ///<May remove later	
 	void                    DoCellLoop()                            ; ///<May remove later
 
-        Bool_t                  isMCprod                                ;///<Flag for MC productions
+        Bool_t                  IsMCprod                                ;///<Flag for MC productions
```

Comment:
If you change a streaming member name, you have to update the LinkDef

Reply:
Even if it's only a name change?Okay I didn't know that, I will update it.

Reply:
I might matter - I'm not 100% sure (since it's the same type, it could very well be fine). But incrementing is cheap, and will for sure avoid any issue :-)

Reply:
Yes that's true, thank you for the feedback. I will keep it in mind for future updates :)